components/Op/OpDetailForm.js

```diff
@@ -353,6 +385,36 @@ class OpDetailForm extends Component {
                     />
                   )}
                 </Form.Item>
+                <Form.Item
+                  label={opSchoolName}
+                  validateStatus={organisationNameError ? 'error' : ''}
+                  help={organisationNameError || ''}
+                >
+                  {getFieldDecorator('organisationName', {
+                    rules: [
+                      {
+                        required: true,
+                        message: 'A School name or organisation must be provided'
+                      }
+                    ]
+                  }
+                  )(<Input placeholder='Name' />)}
+                </Form.Item>
+                <Form.Item
+                  label={opAddress}
+                  validateStatus={organisationAddressError ? 'error' : ''}
+                  help={organisationAddressError || ''}
+                >
+                  {getFieldDecorator('organisationAddress', {
```

Comment:
An address is not mandatory. 
Lots of activities may take place at non specific locations - e.g ONLINE.

If the location select is set to Online - then the address input should not be shown at all.